src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+            yield return new object[] { sbyte.MaxValue, (byte)127 };
+            yield return new object[] { (SByteEnum)0, byte.MinValue };
+            yield return new object[] { (SByteEnum)1, (byte)1 };
+            yield return new object[] { (SByteEnum)1, (byte)1 };
+            yield return new object[] { (SByteEnum)sbyte.MaxValue, (byte)127 };
+
+            // ushort.
+            yield return new object[] { ushort.MinValue, byte.MinValue };
+            yield return new object[] { (ushort)1, (byte)1 };
+            yield return new object[] { (UShortEnum)ushort.MinValue, byte.MinValue };
+            yield return new object[] { (UShortEnum)1, (byte)1 };
+
+            // short.
+            yield return new object[] { (short)0, byte.MinValue };
+            yield return new object[] { (short)1, (byte)1 };
+            yield return new object[] { (ShortEnum)0, byte.MinValue };
+            yield return new object[] { (ShortEnum)1, (byte)1 };
+
+            // uint.
+            yield return new object[] { uint.MinValue, byte.MinValue };
+            yield return new object[] { (uint)1, (byte)1 };
+            yield return new object[] { (UIntEnum)uint.MinValue, byte.MinValue };
+            yield return new object[] { (UIntEnum)1, (byte)1 };
+
+            // int.
+            yield return new object[] { 0, byte.MinValue };
+            yield return new object[] { 1, (byte)1 };
+            yield return new object[] { (IntEnum)0, byte.MinValue };
+            yield return new object[] { (IntEnum)1, (byte)1 };
+
+            // ulong.
+            yield return new object[] { ulong.MinValue, byte.MinValue };
+            yield return new object[] { (ulong)1, (byte)1 };
+            yield return new object[] { (ULongEnum)ulong.MinValue, byte.MinValue };
+            yield return new object[] { (ULongEnum)1, (byte)1 };
+
+            // long.
+            yield return new object[] { (long)0, byte.MinValue };
+            yield return new object[] { (long)1, (byte)1 };
+            yield return new object[] { (LongEnum)0, byte.MinValue };
+            yield return new object[] { (LongEnum)1, (byte)1 };
+
+            // float.
+            yield return new object[] { (float)0, byte.MinValue };
+            yield return new object[] { (float)1, (byte)1 };
+
+            // double.
+            yield return new object[] { (double)0, byte.MinValue };
+            yield return new object[] { (double)1, (byte)1 };
+
+            // decimal.
+            yield return new object[] { (decimal)0, byte.MinValue };
+            yield return new object[] { (decimal)1, (byte)1 };
+
+            // bool.
+            yield return new object[] { true, byte.MaxValue };
+            yield return new object[] { false, byte.MinValue };
+
+            // null.
+            yield return new object[] { null, byte.MinValue };
+
+            // string.
+            yield return new object[] { "0", byte.MinValue };
```

Comment:
Remove this chunk of data and add
```MemberData(nameof(ToByte_String_TestData)]``` to the test below

Reply:
Fixed